cypress/e2e/14-timeline/timeline.spec.ts

```diff
@@ -141,5 +142,34 @@ describe("Timeline", () => {
                 expectAvatar(e, newAvatarUrl);
             });
         });
+
+        it("should display a hidden event line for edited message on IRC layout", () => {
+            sendEvent(roomId);
+            cy.wait(500);
```

Comment:
I doubt that this would be proper way of waiting until the room was ready. Please let me know a better way.